services/frontend-v3/src/components/admin/skillTable/SkillTable.jsx

```diff
@@ -52,36 +54,37 @@ const SkillTable = ({ intl, type }) => {
     } catch (error) {
       // eslint-disable-next-line no-console
       console.log(error);
-      return [];
+      throw error;
     }
   }, [type]);
 
   /* useEffect will run if statement, when the component is mounted */
   /* useEffect will run else statement, if an addition, update/edit or deletion occurs in the table */
   useEffect(() => {
-    let skills = [];
-    let categories = [];
     if (loading) {
       const setState = async () => {
-        skills = await getSkill();
-        categories = await getCategories();
-        setData(skills);
-        setCategories(categories);
+        await getSkill()
+          .then(skills => setData(skills))
+          .then(getCategories)
+          .then(categories => setCategories(categories))
+          .catch(error => handleError(error, "redirect"));
```

Comment:
Not 100% sure if you call multiple async function like that (passing it in `then`), I would either 

```suggestion
        await getSkill()
          .then(skills => setData(skills))
          .catch(error => handleError(error, "redirect"));
        await getCategories()
          .then(categories => setCategories(categories))
          .catch(error => handleError(error, "redirect"));
```

or 

```suggestion
        try {
           setData(await getSkill());
           setCategories(await getCategories());
         } catch (error) {
           handleError(error, "redirect"));
         }
```